src/styles/createMuiTheme.d.ts

```diff
@@ -6,12 +6,18 @@ import { Spacing } from './spacing';
 import { Transitions } from './transitions';
 import { Typography, TypographyOptions } from './createTypography';
 import { ZIndex } from './zIndex';
+import { StyleRules } from './withStyles'
 
 export interface ThemeOptions {
   breakpoints?: Partial<BreakpointsOptions> & Partial<Breakpoints>;
   mixins?: Partial<Mixins>;
   palette?: Partial<Palette>;
   typography?: TypographyOptions | ((palette: Palette) => TypographyOptions);
+  shadows?: Partial<Shadows>;
```

Comment:
I don't think `shadows` should be `Partial`; it's an array of fixed size, and if you specify it here it will be used in its entirety, not merged with defaults.

Reply:
Sorry for it, my mistake